src/haddock/modules/scoring/sasascore/sasascore.py

```diff
@@ -0,0 +1,164 @@
+"""Accessibility scoring calculations."""
+from haddock import log
+from pathlib import Path
+import pandas as pd
+from haddock.core.typing import FilePath
+from haddock.clis.restraints.calc_accessibility import (
+    apply_cutoff,
+    get_accessibility,
+    )
+
+
+def prettify_df(output_fname, columns, sortby=None):
```

Comment:
typing of the function
```suggestion
def prettify_df(output_fname: str, columns: list[str], sortby: Optional[str] = None) -> None:
```